x-pack/plugins/maps/public/components/layer_panel/settings_panel/settings_panel.js

```diff
@@ -18,6 +18,7 @@ import {
   EuiCallOut,
 } from '@elastic/eui';
 import { ValidatedRange } from '../../../shared/components/validated_range';
+import { i18n } from '@kbn/i18n';
```

Comment:
Forget to translate
* Min zoom form row label
* Max zoom form row label
* title `settings`